src/site/_includes/content/lighthouse-pwa/scoring.njk

```diff
@@ -1,9 +1,9 @@
 {% Aside 'note' %}
   Each PWA audit is weighted equally
   in the Lighthouse PWA Score,
-  with the exception of three manual audits:
-  [Site works cross-browser](/pwa-cross-browser),
-  [Page transitions don't feel like they block on the network](/pwa-page-transitions),
-  [Each page has a URL](/pwa-each-page-has-url).
-  Learn more in [The Progressive Web App score](https://developers.google.com/web/tools/lighthouse/v3/scoring#pwa).
+  except for three manual audits:
+  **[Site works cross-browser](/pwa-cross-browser)**,
```

Comment:
I like formatting each audit name in bold. Looks like we're on the same page here.

Reply:
Oh, good. That's actually [specified in the handbook](https://web.dev/handbook/grammar/#ui-elements-and-interaction).